src/main/resources/META-INF/plugin.xml

```diff
@@ -800,6 +800,11 @@
             <add-to-group group-id="NewGroup" anchor="before" relative-to-action="NewFile"/>
         </action>
 
+        <action id="Rust.NewCargoCrate" class="org.rust.ide.actions.RsCreateCrateAction"
+                text="Cargo crate" description="Create new Cargo crate.">
```

Comment:
Stylistics: 
Action names should be headline capitalized (`Cargo Crate`)
Descriptions should not end with `.`

Reply:
Fixed, thanks.